.gitlab-ci.yml

```diff
@@ -13,7 +13,7 @@ stages:
     VERSION: ${CI_COMMIT_REF_SLUG}
     OSTAG: <Tag suffix corresponding to version>
     # Use single quotes to escape colon
-    DESTINATION: '${CI_REGISTRY_IMAGE}/${IMAGE}:${VERSION}-${OSTAG}'
+    DESTINATION: 'ghcr.io/$CI_REGISTRY_USER/${IMAGE}:${VERSION}-${OSTAG}'
```

Comment:
is `CI_REGISTRY_USER` actually a user that is known to github? I think this would need to be `MuonColliderSoft` in this case to end up in the right github registry.